script.js

```diff
@@ -0,0 +1,122 @@
+
+function function1(){
+	if (document.getElementById("square1").textContent === 'X') {
+		document.getElementById("square1").textContent = 'O';
+    document.getElementById("square1").style.color = 'white'
+}
+
+	else { document.getElementById("square1").textContent = 'X'
+       }
+
+};
+
+
+function function2(){
+
+	if (document.getElementById("square2").textContent === 'X') {
+		document.getElementById("square2").textContent = 'O';
+    document.getElementById("square2").style.color = 'white'
+}
+
+	else { document.getElementById("square2").textContent = 'X'
+       }
+
+};
+
+
+function function3(){
+	
+	if (document.getElementById("square3").textContent === 'X') {
+		document.getElementById("square3").textContent = 'O';
+    document.getElementById("square3").style.color = 'white'
+}
+
+	else { document.getElementById("square3").textContent = 'X'
+       }
+
+};
+
+
+function function4(){
+	
+	if (document.getElementById("square4").textContent === 'X') {
+		document.getElementById("square4").textContent = 'O';
+    document.getElementById("square4").style.color = 'white'
+}
+
+	else { document.getElementById("square4").textContent = 'X'
+       }
+
+};
+
+function function5(){
+	
+	if (document.getElementById("square5").textContent === 'X') {
+		document.getElementById("square5").textContent = 'O';
+    document.getElementById("square5").style.color = 'white'
+}
+
+	else { document.getElementById("square5").textContent = 'X'
+       }
+
+};
+
+function function6(){
+	
+	if (document.getElementById("square6").textContent === 'X') {
+		document.getElementById("square6").textContent = 'O';
+    document.getElementById("square6").style.color = 'white'
+}
+
+	else { document.getElementById("square6").textContent = 'X'
+       }
+
+};
+
+function function7(){
+	
+	if (document.getElementById("square7").textContent === 'X') {
+		document.getElementById("square7").textContent = 'O';
+    document.getElementById("square7").style.color = 'white'
+}
+
+	else { document.getElementById("square7").textContent = 'X'
+       }
+
+};
+
+function function8(){
+	
+	if (document.getElementById("square8").textContent === 'X') {
+		document.getElementById("square8").textContent = 'O';
+    document.getElementById("square8").style.color = 'white'
+}
+
+	else { document.getElementById("square8").textContent = 'X'
+       }
+
+};
+
+function function9(){
+	
+	if (document.getElementById("square9").textContent === 'X') {
+		document.getElementById("square9").textContent = 'O';
+    document.getElementById("square9").style.color = 'white'
+}
+
+	else { document.getElementById("square9").textContent = 'X'
+       }
+
+};
```

Comment:
You already mentioned that your code wasn't dry. But, how might we dry this up. If we added an event listener to all elements with, let's say the `square` class. Than you can have this behavior that you described in 9 separate function into one function. And have it execute when that event fires off.